### DISCUSSION TITLE
v3.12.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#9983] [Releases & Announcements]
## Changelog

- Design updates for various setting pages like integrations, canned responses, teams, etc
- Captain integration (alpha)
- Support for shared instagram reels in the messenger channel
- Changes for vue3 upgrade
- Numerous bug fixes and enhancements

###
Thanks to @nestordavalos for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v3.12.0'>v3.12.0</a>.</em>

**Comment by yamlv** [comment]
Tried to do sudo cwctl --upgrade but it installed only [v3.11.1] how to upgrade to latest or any command where i can install specific version? \

Thank you

**Reply by sojan-official** [reply]
you can debug the upgrade using the manual installation steps : https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm#upgrading-to-a-newer-version-of-chatwoot

**Comment by datavatelabs** [comment]
After upgrading, in the front-end UI, showing a notification in corner which says "Disconnected from Chatwoot"

**Reply by sojan-official** [reply]
Check if there are any issues with your web socket connections. Chatwoot requires this connection for real time messaging
